Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/SN3A-Memory-Consumption/memory-consumption

```diff
@@ -0,0 +1,124 @@
+bash Openshift-EE/utils/e2e-cr jobname:memory-consumption-jiva jobphase:Running init_time:"$current_time"
+
+echo "*******Deploying Busybox-sts Application****"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-provision-mem-check-jiva metadata="")
+echo $test_name
+cd litmus
+echo "Running the litmus test for Busybox Deployment.."
+
+cp apps/busybox/deployers/run_litmus_test.yml busybox_memcheck.yml
+
+sed -i -e 's/app: busybox-litmus/app: mem-busybox-jiva/g' \
+-e 's/app=busybox-sts/app=busybox-mem-jiva/g' \ #application label
+-e 's/value: statefulset/value: deployment/g' \
+-e 's/value: openebs-cstor-sparse/value: openebs-jiva-default/g' \
+-e 's/value: app-busybox-ns/value: mem-check-jiva/g' busybox_memcheck.yml
+
+cat busybox_memcheck.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:mem-busybox-jiva' job=busybox_memcheck.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:memory-consumption-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id SN3A 3-functional "Check if the memory consumption doesn't cross the desired threshold for known workload" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+echo "*******Checking Memory Consumption**********"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=memcheck-test-jiva metadata="")
+echo $test_name
+cd litmus
+
+cp apps/prometheus/functional/memcheck/run_litmus_test.yml memcheck_test.yml
+
+sed -i -e 's/name=prometheus/app=busybox-mem-jiva/g' \
+-e 's/prometheus-claim/openebs-busybox/g' \
+-e 's/app: memcheck-cstor-test/app: memcheck-cstor-test-jiva/g' \
+-e 's/value: app-prometheus/value: mem-check-jiva/g' memcheck_test.yml
+
+echo "Running the litmus test for Busybox Deployment Scaleup.."
+cat memcheck_test.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:memcheck-cstor-test-jiva' job=memcheck_test.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:memory-consumption-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id SN3A 3-functional "Check if the memory consumption doesn't cross the desired threshold for known workload" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+echo "********Deprovisioning Busybox Application*******"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-deprovision-mem-check metadata="")
+echo $test_name
+cd litmus
+
+cp apps/busybox/deployers/run_litmus_test.yml deprovision_mem_check.yml
+sed -i -e 's/generateName: litmus-busybox-deploy/generateName: busybox-mem-check-deprovision/g' \
```

Comment:
@harshita-sharma-mayadata generate a different name for the jiva and cStor deprovision test.